src/main/java/io/gravitee/cockpit/api/command/environment/DisableEnvironmentPayload.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.cockpit.api.command.environment;
+
+import io.gravitee.cockpit.api.command.Payload;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class DisableEnvironmentPayload implements Payload {
+
+  private String id;
+
+  private String cockpitId;
+
+  private String name;
+
+  private String userId;
```

Comment:
All of these attributs are required ? Why not defined a constructor with all args ? Instead of build a instance by calling setter by setter

Reply:
I blindly replicated what is done in other payloads 🙈 
Updated using `@Builder`